src/index.js

```diff
@@ -55,3 +62,9 @@ function setCopyrightDate(element) {
     let d = new Date();
     element.html(d.getFullYear().toString());
 }
+
+// Access URL key-value pairs
+// http://stackoverflow.com/questions/11582512/how-to-get-url-parameters-with-javascript/11582513#11582513
+function getURLParameter(name) {
```

Comment:
This function appears to be already defined in the utils.js file. Can we use that instead?

Reply:
Or, is this trying to pick off a single querystring key value pair?

Reply:
I added it here because we aren't including utils.js - but we could - thoughts?

Reply:
Seem to me that we should keep all of the functions that meddle with URL query parameters in one place. At the moment, that looks like utils.js. That file is really small, so there really isn't much of a performance penalty for including it.

Reply:
This should go in utils.js and utils.js should be included for the index.html page.

Reply:
Done.